css/css-animations/parsing/animation-name-invalid.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Animations Level 1: parsing animation-name with invalid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-animations/#propdef-animation-name">
+<meta name="assert" content="animation-name supports only the grammar '[ none | <keyframes-name> ]#'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_invalid_value("animation-name", '12');
```

Comment:
Can we add a test that 'none' is invalid. As per:
"The <custom-ident> additionally excludes the none keyword."
(https://drafts.csswg.org/css-animations/#keyframes)
And CSS-wide keywords like "inherit" and "default": As per:
"The CSS-wide keywords are not valid <custom-ident>s."
(https://drafts.csswg.org/css-values-4/#custom-idents)

Reply:
none is a valid value for the property, it simply isn't a keyword.

For CSS-wide keywords like inherit and initial, I'd prefer a separate test to check that all properties of a spec support them. Like testing inheritance/non-inheritance
https://github.com/web-platform-tests/wpt/pull/12593